streaming/index.js

```diff
@@ -227,6 +227,19 @@ const startWorker = (workerId) => {
   };
 
   const wsVerifyClient = (info, cb) => {
+    const PUBLIC_STREAMS = [
```

Comment:
I think that it is better to define PUBLIC_STREAMS and PUBLIC_ENDPOINTS outside the function.